src/components/SideBar/DesktopSideBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { FoldIcon, LogoIcon } from '@assets';
+import DesktopSideBarContents from './DesktopSideBarContents';
+
+interface SideBarProps {
+  isOpen: boolean;
+  width: number;
+  toggleSideBar: () => void;
+}
+
+function DesktopSideBar({ isOpen, width, toggleSideBar }: SideBarProps) {
+  return (
+    <>
+      {isOpen && width <= 1920 && (
+        <div
+          className="absolute left-0 top-0 h-dvh w-dvw bg-black opacity-50"
+          onClick={() => toggleSideBar()}
```

Comment:
[Suggestion] props 없으면 그냥 이렇게 써도 될거에용
```suggestion
          onClick={toggleSideBar}
```